modules/ROOT/pages/migration/index.adoc

```diff
@@ -1,9 +1,9 @@
-[[migration-guide]]
-:description: This page lists the breaking changes from version 3.0.0 to 4.0.0 and describes how to update.
-= Migration to 4.0.0
-:page-aliases: guides/index.adoc, guides/migration-guide/index.adoc, guides/migration-guide/server.adoc, guides/migration-guide/type-definitions.adoc, guides/migration-guide/mutations.adoc
+[[v5-migration]]
+:description: This page lists the breaking changes from version 4.0.0 to 5.0.0 and describes how to update.
+:page-aliases: guides/v5-migration/index.adoc , migration/v5-migration/index.adoc
```

Comment:
The page-aliases work in a way that, let's say, when a user goes to www.website.com/oldpage, instead of getting a 404 error, they will be redirected to www.website.com/newpage. For that, the alias needs to be added in the newpage. TLDR, we should bring the page-aliases from the old 4.0.0 migration guide here

```suggestion
:page-aliases: guides/index.adoc, guides/migration-guide/index.adoc, guides/migration-guide/server.adoc, guides/migration-guide/type-definitions.adoc, guides/migration-guide/mutations.adoc
```